plenum/common/messages/client_request.py

```diff
@@ -62,6 +76,17 @@ class ClientNYMOperation(MessageValidator):
         # do roles list expandable form outer context
     )
 
+    def _validate_message(self, dct):
+        # TODO other cases:
+        #   - full verkey | short/long target nym
+        #   - omitted verkey | short/long target nym
+        verkey_abbreviated = not AbbreviatedVerkeyField().validate(dct[VERKEY])
```

Comment:
Here we call static validation (`AbbreviatedVerkeyField`) and base 58 decoding twice. 
Previously we found that static validation is one of the bottle necks. So, does additional validation and decoding affect performance here? Maybe we need to think how we can avoid it.